src/alexandria3k/data_sources/uspto.py

```diff
@@ -21,21 +21,25 @@
 import os
 import re
 
+from alexandria3k.common import fail
```

Comment:
`fail` is no longer available. Instead we raise exceptions. Please make sure you have rebased your changes on top of the current main version.